themes/bootstrap3/templates/layout/js-configs.phtml

```diff
@@ -7,4 +7,13 @@
 //$this->jsConfigs()->add('myConfig', true);
 //$this->jsConfigs()->add('siteTitle', $this->config()->get('config')->Site->title);
 
+$this->jsConfigs()->add(
+    'item-status:load-batch-wise',
+    $this->config()->get('config')->Item_Status->load_batch_wise
```

Comment:
We should probably add default values here in case these two settings are missing.

Reply:
Done